src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -268,10 +268,10 @@ public function testDecodeFloatAttribute()
     {
         $source = <<<XML
 <?xml version="1.0"?>
-<document index="-12.11">Name</document>
```

Comment:
Let's not change this test. But feel free to add assertions for `12.11` and `0.123` in the `testDecodeFloatAttribute`

Reply:
I've decided to change it because this test was exactly the same as the next one, `testDecodeNegativeFloatAttribute`, i.e. the same values.

So do you suggest keeping `-12.11` here, adding `12.11` and `0.123` in this xml, **and** still keeping the `testDecodeNegativeFloatAttribute` method (with `-12.11` repeated)?
Fine with me, I just want to clarify it.